### ISSUE TITLE
Please advise how to get this running from installing in from within Umbraco package installer.

### ISSUE THREAD

**Issue by kometa333** [skttl/umbraco-fulltextsearch8#3]
I installed the package, I can see in the back-office that the search is working. But I'm unable to include it in my Search page in the website. Are you able to point me in the right direction?

Thanks.

**Comment by skttl** [comment]
Did you see this?
https://github.com/skttl/Our.Umbraco.FullTextSearch/blob/master/docs/developers-guide.md

**Comment by kometa333** [comment]
Yes, I did. But I can't get the model carry over any values from the controller to the .mvc page. The way I've implemented it is:
I've created a controller in the App_code folder:

using System.Web.Mvc;
using Our.Umbraco.FullTextSearch.Interfaces;
using Our.Umbraco.FullTextSearch.Models;
using Umbraco.Web;
using Umbraco.Web.Models;
using Umbraco.Web.Mvc;

namespace MyProject.App_Code.Controllers
{
    public class SearchController : RenderMvcController
    {
        private readonly ISearchService _searchService;

        public SearchController(ISearchService searchService)
        {
            _searchService = searchService;

        }

        public override ActionResult Index(ContentModel model)
        {
            if (Request["q"] != null)
            {
                int.TryParse(Request["p"], out int currentPage);
                currentPage = currentPage < 1 ? 1 : currentPage;

                var search = new Search(Request["q"])
                    .SetCulture(model.Content.GetCultureFromDomains().ToLower());

                ViewBag.FullTextSearchResult = _searchService.Search(search, currentPage);
            }
            return base.Index(model);
        }
    }
}

Then I've added this in the Search.cshtml file:
@if (ViewBag.FullTextSearchResult != null)
{
    <ul>
        @foreach (var result in ViewBag.FullTextSearchResult.Results)
        {
            <li>
                <a href="@result.Url()">@result.Title</a>
                <p>
                    @result.Summary
                </p>
                <small>
                    Url: @result.Url()<br />
                    Id: @result.Id<br />
                    Last updated: @(result.Content()?.UpdateDate)<br />
                    Score: @result.Score
                </small>
            </li>
        }
    </ul>

    <div>Total results: @ViewBag.FullTextSearchResult.TotalResults</div>

    if (ViewBag.FullTextSearchResultt.CurrentPage > 1)
    {
        <a href="?q=@Request["q"]&p=@(ViewBag.FullTextSearchResult.CurrentPage-1)">Previous page</a>
    }

    if (ViewBag.FullTextSearchResult.CurrentPage < ViewBag.FullTextSearchResult.TotalPages)
    {
        <a href="?q=@Request["q"]&p=@(ViewBag.FullTextSearchResult.CurrentPage+1)">Next page</a>
    }
}

But as the "ViewBag.FullTextSearchResult" is always null, nothing is returned. Even though the "q" query sting has a value.

Can you please see what is done wrong here?

Thanks.

**Comment by skttl** [comment]
Seems right, can you try to step through the Index method to see if the if condition is met?

**Comment by kometa333** [comment]
When I place the site into debug it will not step through to the Index method.

**Comment by kometa333** [comment]
Hi, just to update you. I manged to step trough into the Index method and the " ViewBag.FullTextSearchResult" bring back "Enumeration yielded no results". Even though when I call back the search results using: "Umbraco.ContentQuery.Search(Request["q"], string.Empty)", it brings back several results.

**Comment by skttl** [comment]
Can you inspect the nodes via the examine dashboard? You should have content in the full text search fields (check the docs for reference).

If no content is present in full text search fields, you can't get any results.

**Comment by kometa333** [comment]
Also, one other thing which I've just spotted that is not working properly. I've added a property called "hideInSearchResults" with boolean value for each page of the site and have added this property in the web.config like this:
<add key="FullTextSearch.DisallowedPropertyAliases" value="hideInSearchResults"/>

However, when I search in the cms back-office the page which has this tick box checked still comes back in the results.

Are you able to get back to me? I'd like to know if your search can be fixed and used on our site or shall I look for a different solution.

Thanks.
